src/Workspaces/CoreTestUtilities/Roslyn.Services.UnitTests.Utilities.csproj

```diff
@@ -4,7 +4,7 @@
   <PropertyGroup>
     <OutputType>Library</OutputType>
     <RootNamespace>Microsoft.CodeAnalysis.UnitTests</RootNamespace>
-    <TargetFrameworks>netcoreapp3.1;net472</TargetFrameworks>
+    <TargetFrameworks>net5.0;netcoreapp3.1;net472</TargetFrameworks>
```

Comment:
I didn't understand why some projects add "net5.0" (keeping "netcoreapp3.1"), but others replace "netcoreapp3.1". What's the rationale?

Reply:
All of the projects which replaced `netcoreapp3.1` with `net5.0` were unit test projects. The change reflected the desire to compile and run the tests against `net5.0`. This particular assembly is a utility assembly and used by compiler and IDE unit tests. 

As of this PR there are no IDE unit tests that target `netcoreapp3.1` however there are PRs to enable exactly this. Leaving the `netcoreapp3.1` here to support those PRs.